src/components/CalendarComponent/CalendarComponent.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState, setState, useEffect } from "react";
+import Calendar from "react-calendar/dist/entry.nostyle";
+import "CalendarComponent/CalendarComponent.css";
+import { withFirebase } from "utils/Firebase";
+function CalendarComponent(props) {
```

Comment:
hmm could you rename this to Calendar or DateSelector, just following along with Jason's naming standards